pkg/kubeutil/client.go

```diff
@@ -28,6 +28,9 @@ import (
 	"k8s.io/client-go/rest"
 	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
 	csidriver "sigs.k8s.io/secrets-store-csi-driver/apis/v1alpha1"
+
+	// Import kubernetes auth plugins
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
```

Comment:
Note: @superbeeny ran into an issue where UCP isn't doing auth to Kubernetes correctly for OIDC. This is the fix.